tests/test_endf.py

```diff
@@ -921,9 +921,9 @@ def test_evaluation_neutron():
     assert u235.info["sublibrary"] == 10
     assert u235.info["format"] == 6
     assert u235.material == 9228
-    assert u235.projectile["mass"] == 1.0
+    assert_allclose(u235.projectile["mass"], 1.0)
```

Comment:
I'm concerned about these changes.  I'm definitely curious to see how the `endftod` result is different from the exact test.  I know that there can be floating point issues with exact comparison, but I hoped that conversion from string would not be subject to hose kinds of issues.